src/base/AsyncJob.cc

```diff
@@ -179,3 +188,26 @@ const char *AsyncJob::status() const
     return buf.content();
 }
 
+void
+AsyncJob::ReportAllJobs(StoreEntry *e)
+{
+    PackableStream os(*e);
+    // this loop uses YAML syntax, but AsyncJob::status() still needs to be adjusted to use YAML
+    const char *indent = "    ";
+    for (const auto job: Jobs_) {
+        os << indent << job->id << ":\n";
+        os << indent << indent << "type: '" << job->typeName << "'\n";
+        os << indent << indent << "status:" << job->status() << '\n';
+        if (!job->started_)
+            os << indent << indent << "started: false\n";
+        if (job->stopReason)
+            os << indent << indent << "stopped: '" << job->stopReason << "'\n";
```

Comment:
Reason text may contain single-quote characters, yes? which break this quoting style.
There are two alternatives;

Either use double-quotes and ensure that reason text is appropriately slash-escaped per YAML:
```suggestion
            os << indent << indent << "stopped: \"" << Format::QuoteYamlString(job->stopReason) << "\"\n";
```
(this will require a YAML specific encoder added to `src/format/Quoting.cc` until we add a dedicated YAML library).

or, use the YAML "plain style" lines:
```suggestion
            os << indent << indent << "stopped: " << job->stopReason << "\n";
```

Reply:
Punted to future PR.